Mathlib/Topology/ContinuousFunction/StoneWeierstrass.lean

```diff
@@ -378,6 +378,7 @@ theorem Subalgebra.SeparatesPoints.isROrC_to_real {A : StarSubalgebra 𝕜 C(X,
 
 variable [CompactSpace X]
 
+set_option synthInstance.maxHeartbeats 30000 in
```

Comment:
The proof is slightly slower with `Type*` than with `Type _`, although the statements are the same in the end...